tests/client_generators/input_types_generator/test_names.py

```diff
@@ -137,3 +138,26 @@ def test_generate_returns_module_with_fields_names_converted_to_snake_case(
 
     class_def = get_class_def(module)
     assert compare_ast(class_def, expected_class_def)
+
+
+def test_generate_returns_module_with_valid_field_names():
+    schema = """
+    input KeywordInput {
+        in: String!
+        from: String!
+        and: String!
+    }
+    """
+
+    generator = InputTypesGenerator(
+        schema=build_ast_schema(parse(schema)), enums_module="enums"
+    )
+
+    module = generator.generate()
+
+    parsed = ast.parse(
+        ast_to_str(module)
+    )  # Round trip because invalid identifiers get picked up in parse
+    class_def = get_class_def(parsed)
+    field_names = get_assignment_target_names(class_def)
+    assert field_names == {"in_", "from_", "and_"}
```

Comment:
Format code with black.

Reply:
This has been formatted with black. What differences are you expecting?

Reply:
That's weird. I would swear that black would add empty trailing line to the file? 🤔

Reply:
The file ends in a line feed character (I downloaded it from here and checked) but I don't think that github displays it. Every file when viewed in github has no trailing empty line.